networking/src/Models/IQuestionTemplate.ts

```diff
@@ -0,0 +1,18 @@
+import { IGameTemplate } from './IGameTemplate';
+
+export interface IQuestionTemplate {
+  id: string,
+  title?: string | null,
+  owner?: string | null,
+  version?: number | null,
+  choices?: string | null,
+  instructions?: string | null,
+  domain?: string | null | undefined,
+  cluster?: string | null | undefined,
+  grade?: string | null | undefined,
+  standard?: string | null | undefined,
+  imageUrl?: string | null | undefined,
+  gameTemplates?: IGameTemplate[] | null,
+  createdAt?: string | null | undefined,
+  updatedAt?: string | null
```

Comment:
These have been set to nonnull and typed to `Date`. These are received from AWS as a `string` so we are handling the conversion in the `ApiClient` when we parse either the game or the question templates. We are relying on the fact that AWS will provide standardized ISO 8601 DateTime formatted strings to avoid browser inconsistency issues.